pymc3/distributions/continuous.py

```diff
@@ -1554,6 +1554,7 @@ def __init__(self, alpha, beta, *args, **kwargs):
         self.median = beta * tt.exp(gammaln(tt.log(2)))**(1. / alpha)
         self.variance = (beta**2) * \
             tt.exp(gammaln(1 + 2. / alpha - self.mean**2))
+        self.mode = tt.switch(self.alpha >= 1, self.beta * ((self.alpha - 1)/self.alpha) ** (1 / self.alpha) , 0)  # Reference: https://en.wikipedia.org/wiki/Weibull_distribution
```

Comment:
line-breaks would be helpful.

Reply:
Also, is it better to use `alpha` or `self.alpha`? I think I should change them back to `self.alpha`, otherwise it's inconsistent with the rest of the code.